src/content/docs/en/guides/rss.mdx

```diff
@@ -93,7 +95,7 @@ export async function get(context) {
 }
 ```
 
-You can configure your collection schema to enforce these expected RSS properties. Import and apply `rssSchema` to ensure that each collection entry produces a valid RSS feed item.
+Configure your collection schema to enforce these expected RSS properties. Import and apply `rssSchema` to ensure that each collection entry produces a valid RSS feed item.
```

Comment:
```suggestion
Optional: replace your existing blog collection schema to enforce the expected RSS properties.

To ensure that every blog entry produces a valid RSS feed item, you can optionally import and apply `rssSchema` instead of defining each individual property of your schema.
```
I think this is key here. Someone will probably already have their blog collection schema defined, so we should be clear that this is an optional refactoring they can do so that the RSS helper actually defines the schema, instead of listing out each property using `z.` etc.